### PR TITLE
(API) Salesforce Marketing Cloud / Exact Target destination: Support for external audiences

### PR DESCRIPTION
**Pull request by mohan-h**
_*** Edited incorrect title and description ***_

Functionality update

You can now activate external audiences originating from custom uploads to this destination. See the [supported audiences](#supported-audiences) section for more details.

### CODE REVIEW

File: help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md [hunk 1]
```diff
@@ -300,6 +300,7 @@ This section captures the functionality and significant documentation updates ma
 
 |Release month|Update type|Description|
 |---|---|---|
+|November 2023|Functionality update|<ul><li>We now automatically convert the **[UICONTROL Mapping ID]** values in the [schedule audience export](#schedule-segment-export-example) step to `lowercase`. Additionally we also sanitize `spaces` and `special characters`. This is done to ensure compliance with [!DNL Salesforce Marketing Cloud] field names.</li><li>We now support custom uploads, see the [supported audiences](#supported-audiences) section.</li></ul>|
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md]: @mohan-h  - in addition to @vfalon 's comments on the other two PRs, please note that you need to replace: 

`This is done to ensure compliance with [!DNL Salesforce Marketing Cloud] field names` with Hubspot.

Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md]: according to the documentation that the team created for Hubspot, there is no Mapping ID for this connector. - https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/crm/hubspot.html?lang=en

Yet here we mention **[UICONTROL Mapping ID]** values. Can we please explore and correct this difference?

**Review by vgiurgiu — CHANGES_REQUESTED** [review]
thanks for the update, @mohan-h - please see my comments and LMK of any questions.

File: help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md (hunk 1 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md]: Apologies, kindly re-review seems I mixed up the PR title and comments

**Comment by vfalon** [comment]
Closing this PR for now. We'll reopen it if we decide to implement this functionality.

**Pull request closed without merging**